monkey/monkey_island/cc/ui/src/components/pages/RunMonkeyPage/utils/ServerUtils.tsx

```diff
@@ -0,0 +1,5 @@
+function getNodePort() {
+    return process.env.JAVASCRIPT_RUNTIME_PORT;
+}
+
+export const NODE_PORT = getNodePort();
```

Comment:
Use this in `monkey_island/cc/next_ui/server.js`. Also, maybe use `FRONT_END_PORT` or `JS_RUNTIME_PORT` as we could migrate to bun from node

Reply:
@shreyamalviya this was not resolved. Since this is UI refactoring branch, we don't necessarily care as much about modifying the legacy UI

Reply:
Fixed now